### PR TITLE
docs: add create bot documentation

### PR DESCRIPTION
**Pull request by MaximeKoitsalu**

### CODE REVIEW

File: CREATE_BOT.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+So you’ve made a Bot User but it’s not actually in any server.
+
+If you want to invite your bot you must create an invite URL for it.
+
+1. Make sure you're logged on to the [Discord website](https://discord.com/)
+2. Navigate to the [application page](https://discord.com/developers/applications)
+3. Click on your bot's page.
+4. Go to the "OAuth2" tab.
+5. Tick the “bot” checkbox under “scopes”.
+6. Tick the permissions required for your bot to function under “Bot Permissions”.
```
Comment (milan090) [inline CREATE_BOT.md]: Do mention the required permissions required for this particular bot that is connect, speak, send message, view channels, add reaction (for future use)

**Review by milan090 — CHANGES_REQUESTED** [review]
Do mention the required permissions required for this particular bot that is connect, speak, send message, view channels, add reaction (for future use)

**Comment by MaximeKoitsalu** [comment]
@milan090 added your requested change :)! cool bot by the way, will recommend it to my friends 💯

**Comment by milan090** [comment]
Great work and thank you for your contribution :+1:

**Pull request merged**